src/titiler/application/titiler/application/main.py

```diff
@@ -122,6 +123,7 @@ def validate_access_token(access_token: str = Security(api_key_query)):
         router_prefix="/stac",
         extensions=[
             stacViewerExtension(),
+            renderExtension(),
```

Comment:
I think we should rename it `stacRenderExtension`

Reply:
I submitted it  3 minutes ago, how are you so fast? 🙃 

Makes perfect sense, I will rename it